cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -280,12 +280,10 @@ spec:
         - name: ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS
           value: {{ .Values.unreachableNodeTolerationSeconds | quote }}
 {{- end }}
-{{- if .Values.allowMultipleFilesystems }}
-        - name: ROOK_ALLOW_MULTIPLE_FILESYSTEMS
```

Comment:
I think we need to keep this for now, otherwise helm users won't have a way to use multiple filesystems unless they move to Pacific. Or is your point that we should force them to use Pacific where it's actually supported? Maybe I'm trying too hard to support backward compatibility of an experimental feature. :)

Reply:
The goal is to "hide" this option not until we remove it in 1.7, I can keep it for the Helm if necessary.

Reply:
Ok, how about if we just add the helm setting back